pkg/bootstrap/bootstrap.go

```diff
@@ -140,6 +144,44 @@ func (c *BootstrapClient) Bootstrap(ctx context.Context, options BootstrapOption
 	c.handleTelemetry(options.DisableTelemetry, elapsed)
 
 	statusMessage(fmt.Sprintf("Glasskube successfully installed! (took %v)", elapsed.Round(time.Second)), true)
+	if output != "" {
+		if err := convertAndPrintManifests(manifests, output); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func convertAndPrintManifests(
+	objs []unstructured.Unstructured,
+	output string,
+) error {
+	scheme := runtime.NewScheme()
+	var opt bool
+	var div string
+	switch output {
+	case "json":
+		opt = false
+		div = ","
+	case "yaml":
+		opt = true
+		div = "---"
+	}
+	serializer := js.NewSerializerWithOptions(
+		js.DefaultMetaFactory, scheme, scheme,
+		js.SerializerOptions{Yaml: opt, Pretty: true, Strict: true},
+	)
```

Comment:
We don't usually use the `serializer` package, instead we use the `json` an `yaml` packages directly. Please use those instead. Also, please make sure that the format is consistent with the other commands: JSON output should be printed as a JSON array, while YAML output should be printed in YAML multi document mode (like you already do).